otlp/otlp_grpc_metrics_client.h

```diff
@@ -18,24 +18,59 @@
 
 #include <config.h>
 
-#include <google/protobuf/util/json_util.h>
 #include <grpcpp/grpcpp.h>
 #include <opentelemetry/proto/collector/metrics/v1/metrics_service.grpc.pb.h>
 
+#include <platform/types.h>
```

Comment:
Since this is our "library", this include should be moved before third-party library includes (grpc, opentelemetry).